JCDispatch/src/main/java/com/openlattice/integrations/jcDispatch/Dispatch.java

```diff
@@ -0,0 +1,577 @@
+
+        logger.info( "Using the following idToken: Bearer {}", jwtToken );
+
+        Retrofit retrofit = RetrofitFactory.newClient( environment, () -> jwtToken );
+        EdmApi edmApi = retrofit.create( EdmApi.class );
+        PermissionsApi permissionApi = retrofit.create( PermissionsApi.class );
+
+        Dataset<Row> person = sparkSession
+                .read()
+                .format( "com.databricks.spark.csv" )
+                .option( "header", "true" )
+                .load( personPath );
+
+        Dataset<Row> Unit = sparkSession
+                .read()
+                .format( "com.databricks.spark.csv" )
+                .option( "header", "true" )
+                .load( unitPath );
+
+        Dataset<Row> dispatch = sparkSession
+                .read()
+                .format( "com.databricks.spark.csv" )
+                .option( "header", "true" )
+                .load( dispatchPath );
+
+        Dataset<Row> disType = sparkSession
+                .read()
+                .format( "com.databricks.spark.csv" )
+                .option( "header", "true" )
+                .load( disTypePath );
+
+        RequiredEdmElements requiredEdmElements = ConfigurationService.StaticLoader
+                .loadConfiguration( RequiredEdmElements.class );
+
+        if ( requiredEdmElements != null ) {
+            RequiredEdmElementsManager manager = new RequiredEdmElementsManager( edmApi, permissionApi );
+            manager.ensureEdmElementsExist( requiredEdmElements );
+        }
+
+        logger.info( "ER Field names: {}", Arrays.asList( person.schema().fieldNames() ) );
+        logger.info( "ER Field names: {}", Arrays.asList( Unit.schema().fieldNames() ) );
+        logger.info( "ER Field names: {}", Arrays.asList( dispatch.schema().fieldNames() ) );
+        logger.info( "ER Field names: {}", Arrays.asList( disType.schema().fieldNames() ) );
+
+
+        Flight UnitMapping = Flight
+                .newFlight()
+                .createEntities()
+                .addEntity("Unit")
+                .to("Unit")
+                .useCurrentSync()
+                .addProperty("general.unitID", "officerid")
+                .addProperty("place.originatingAgencyIdentifier", "ori")
+                .addProperty("general.active")
+                .value( row -> getEmployeeId( row.getAs( "employeeid" ) ) ).ok()
+                .addProperty("person.employeeID")
+                .value( row -> getActive( row.getAs( "employeeid" ) ) ).ok()
+                .addProperty("person.title", "Title")
+                .addProperty("nc.PersonGivenName")
+                .value( row -> formatText( row.getAs( "FirstName" ) ) ).ok()
+                .addProperty("nc.PersonSurName")
+                .value( row -> formatText( row.getAs( "LastName" ) ) ).ok()
+                .endEntity()
+                .endEntities()
+                .done();
+
+
+        Flight dispatchMapping = Flight
```

Comment:
Please properly indent the chained flight methods; take a look at `DispatchFlight.java` as a reference on how to indent the various levels of the chain. Your code should look like this:

```
Flight dispatchMapping = Flight
  .newFlight()
    .createEntities()
      .addEntity("CallForService")
        .to("CallForService")
        .useCurrentSync()
        .addProperty("general.cfsID", "Dis_ID")
        .addProperty("date.receivedDatetime")
          .value( row -> dateHelper0.parse( row.getAs( "CFS_DateTimeJanet" ) ) )
          .ok()
        .addProperty(...)
        ...
        .endEntity()
      .endEntities()
    .createAssociations()
      ...
      .endAssociations()
    .done()
```


You'll also need to turn off turn off the IntelliJ formatter by surrounding the entire flight with a comment like this:

```
// @formatter:off
Flight dispatchMapping = Flight
  ...
// @formatter:on
```